src/Bem2d.jl

```diff
@@ -26,17 +26,81 @@ function obsgrid(xmin, ymin, xmax, ymax, npts)
     return xobs[:], yobs[:]
 end
 
+export Element
+struct Element
```

Comment:
Note that this can now be immutable!  This is a huge benefit for performance in general.

Reply:
See also [StructArrays](https://github.com/JuliaArrays/StructArrays.jl) for converting back and forth between arrays-of-structs vs. structs-of-arrays.